script/Ciel/Day 13/13_01_02_CIEL13_7.txt

```diff
@@ -31,37 +31,37 @@ But then again, as long as my left arm isn't fully recovered, and I continue to
 [sha:98a2a9577c6f2b1675cead32d9b2fb04fa7ca259]{
 -- Page 203, Offset 35844.
 -- だが、おかげで意識があるうちは戦えた。
-But thanks to that, I was able to fight while I'm conscious.
+But thanks to that, I've been able to fight while remaining conscious.
 }
 [sha:750f362694a8884f4b75a655bf27c8a6425a9698]{
 -- Page 203, Offset 35845.
 -- これで40体。たしか残りは10体程度。
-With this, that's forty bodies down. There should ten remaining if I'm not mistaken.
+With this, that's forty of them down. There should be around ten left.
 }
 [sha:671bf18f004b477e0e11b339e5b369c34a950224]{
 -- Page 204, Offset 35846.
 -- 頭痛はもう気にならない。それより全身にかかる負荷で内側から破裂しそうだ。
-The headache doesn't bother me anymore. But more importantly, I feel like I'll explode from the inside from all strain in my entire body.
+The headache isn't bothering me anymore. More importantly, I feel like my entire body is about to rupture because of all the strain I'm putting on it.
 }
 [sha:aad3827b1b79c8c1715f2a17aa529ab6b04ddd34]{
 -- Page 204, Offset 35847.
 -- ―――でもいける、まだ戦える。
-―――But still, I can move. I can still fight.
+―――But I can still move. I can still fight.
 }
 [sha:d74298c2d4702d5a88064e7700414db702d38b64]{
 -- Page 204, Offset 35848.
 -- うまくすればすべての死者を片付けて、あの吸血鬼をもう一度引きずり下ろせる可能性が、
-If things go accordingly, I can clean up the rest of the Dead, and then there's the possibility that I can drag that vampire back down once again...
+If things go according to plan, I'll be able to clean up the rest of the Dead, and then I'll knock her right off her perch―――
 }
 [sha:03faa911183ebf91b7aefae2490c68a5e758293c]{
 -- Page 205, Offset 35849.
 -- 「いい顔ね。終わりが見えてきて生き返ったって感じ？
-"That's a nice look you got there. Are you feeling alive again after seeing the end of the tunnel?
+"You've got a nice look on your face. Perhaps this is one of those cases where people come back to life after seeing the light?
 }
 [sha:9ed1abd420b569eac4745d09a3e94c997e30ce3b]{
 -- Page 205, Offset 35850.
 -- ええ、実際アナタは凄いわ志貴クン。わたし感動しちゃった。だからご褒美をあげる。―――その希望をグチャグチャに踏み潰して、思いっきり泣かせてあげるわ」
-Yeah, you really are amazing, Shiki-kun. I'm impressed. That's why I'll give you a reward. ―――I'll trample down on that hope of yours, and make you weep with all my heart!"
+Yeah, you really are amazing, Shiki-kun. I'm impressed. That's why I'll give you a reward. ―――I'll trample on that hope of yours, and make you weep with all my heart!"
```

Comment:
"make you weep with all my heart" makes it sound like he's gonna weep using her emotions
> I'll trample on that hope of yours and make you weep to my heart's content!